gammapy/estimators/excess_map.py

```diff
@@ -67,10 +67,6 @@ class ExcessMapEstimator(Estimator):
     If a model is set on the dataset the excess map estimator will compute the excess taking into account
     the predicted counts of the model.
 
-    Some background estimation techniques like ring background or adaptive ring background will provide already
```

Comment:
Replace with a statement / warning on the fact that it is correlated twice? Whatever this does to the statistics...